numpydoc/docscrape.py

```diff
@@ -236,10 +236,30 @@ def _parse_param_list(self, content):
 
         return params
 
+    _role = r":(?P<role>\w+):"
+    _funcbacktick = r"`(?P<name>(?:~\w+\.)?[a-zA-Z0-9_.-]+)`"
+    _funcplain = r"(?P<name2>[a-zA-Z0-9_.-]+)"
+    _funcname = r"(" + _role + _funcbacktick + r"|" + _funcplain + r")"
+    _funcnamenext = _funcname.replace('role', 'rolenext').replace('name', 'namenext')
+    _description = r"(?P<description>\s*:(\s+(?P<desc>\S+.*))?)?\s*$"
+    _func_rgx = re.compile(r"^\s*" + _funcname + r"\s*", re.X)
+    _line_rgx = re.compile(
+        r"^\s*"
+        + r"(?P<allfuncs>"          # group for all function names
+        + _funcname
+        + r"(?P<morefuncs>([,]\s+"
+        + _funcnamenext + r")*)"
+        + r")"                      # end of "allfuncs"
+        + r"(\s*,)?"                # Some function lists have a trailing comma
+        + _description,
+        re.X)
+
     _name_rgx = re.compile(r"^\s*(:(?P<role>\w+):"
                            r"`(?P<name>(?:~\w+\.)?[a-zA-Z0-9_.-]+)`|"
                            r" (?P<name2>[a-zA-Z0-9_.-]+))\s*", re.X)
 
+    empty_description = '..'
```

Comment:
I don't get why this is a public attribugte

Reply:
It is (now) used in `test_docscrape.py:line_by_line_compare`.